evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -575,13 +506,16 @@ contract FluxAggregator is AggregatorInterface, AggregatorV3Interface, Owned {
    */
   function requestNewRound()
     external
+    returns (uint32)
   {
     require(requesters[msg.sender].authorized, "not authorized requester");
 
     uint32 current = reportingRoundId;
     require(rounds[current].updatedAt > 0 || timedOut(current), "prev round must be supersedable");
 
-    requesterInitializeNewRound(current.add(1));
+    uint32 newRoundId = current.add(1);
+    requesterInitializeNewRound(newRoundId);
+    return newRoundId;
```

Comment:
Can we cast this to a `uint80` since that's what users must use to query `getRoundData`?